src/WSSlots.php

```diff
@@ -166,14 +215,36 @@ final public static function editSlots(
 		$flags = EDIT_INTERNAL;
 		$comment = CommentStoreComment::newUnsavedComment( $summary );
 
-		if ( $watchlist === "nochange" ) {
```

Comment:
Preferably there still is the option to add EDIT_SUPPRESS_RC for other types of edits.